app/templates/content/framework-crown-hosting.html

```diff
@@ -0,0 +1,276 @@
+    </ul>
+    <h3>
+      Connectivity
+    </h3>
+    <p>
+      The datacentres are connected to the:
+    </p>
+    <ul>
+      <li>
+        Public Services Network (PSN)
+      </li>
+      <li>
+        NHS Network (N3)
+      </li>
+      <li>
+        Scalable Link Interface (SLI) networks
+      </li>
+      <li>
+        Restricted LAN Interconnect (RLI) networks
+      </li>
+      <li>
+        internet
+      </li>
+    </ul>
+    <h3>
+      Service-level agreements
+    </h2>
+    <p>
+      The service contract covers:
+    </p>
+    <ul>
+      <li>
+        power uptime
+      </li>
+      <li>
+        power outages
+      </li>
+      <li>
+        environmental performance
+      </li>
+      <li>
+        WAN uptime (if purchased)
+      </li>
+      <li>
+        incident management
+      </li>
+      <li>
+        change management
+      </li>
+      <li>
+        capacity management
+      </li>
+      <li>
+        latency (if purchased)
+      </li>
+      <li>
+        backup and disaster recovery through dual-datacentre campuses
+      </li>
+    </ul>
+    <p>
+      Specific government requirements (eg those relating to latency or service levels) can be agreed with the supplier and framework authority.
+    </p>
+
+    <h3>
+      Infrastructure management and remote support
+    </h3>
+    <p>
+      Datacentre infrastructure management tools offer the government real-time insight into:
+    </p>
+    <ul>
+      <li>
+        power usage
+      </li>
+      <li>
+        cooling
+      </li>
+      <li>
+        asset location
+      </li>
+      <li>
+        infrastructure availability
+      </li>
+    </ul>
+    <p>
+      The government will get remote support for certain tasks, eg reboots and power cycling.
+    </p>
+
+    <h3>
+      Quality and environmental standards
+    </h3>
+    <p>
+      The datacentres are compliant with the:
+    </p>
+    <ul>
+      <li>
+        ISO 9001 standard for quality management
+      </li>
+      <li>
+        ISO 14001 standard for environmental management
+      </li>
+      <li>
+        EU Code of Conduct, which is aligned with industry-recognised energy efficiency practices
+      </li>
+    </ul>
+
+
+
+    <h2 id="further-information">
+    <span class="number">4. </span>Further information </h2>
+    <p>
+      Crown Hosting Data Centre services have been available to central government, public sector organisations and arm’s length bodies since March 2015. If you want to learn more about the framework, please contact <a href="mailto:crownhosting@cabinetoffice.gov.uk">crownhosting@cabinetoffice.gov.uk</a>.
+    </p>
+    <p>
+      If you’d like to buy technology services from another framework, please email <a href="enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or <a rel="external" href="http://ccs-agreements.cabinetoffice.gov.uk">find an agreement</a> on the Crown Commercial Service (CCS) website.
```

Comment:
This should have `mailto` as protocol - at the moment it links to `http://localhost:5002/crown-hosting/enquiries@digitalmarketplace.service.gov.uk`